app/views/embed/_metadata_panel.html.erb

```diff
@@ -28,8 +28,7 @@
         <% if viewer.purl_object.license.present? %>
           <dt>License</dt>
           <dd>
-            <span class='sul-embed-license-<%= viewer.purl_object.license[:machine] %>'></span>
```

Comment:
Just calling this out. It's OK to move away from these spans and the "machine" biz entirely at this point, yeah?

Reply:
I assume they just drive the badgy things? I don't know if there are some css classes we get to drop?

Reply:
@cbeer I searched around and found none of these `sul-embed-license-*` CSS classes but perhaps I just missed them or they're dynamic or something?

Reply:
🤷‍♂️